share/pmacc/examples/gameOfLife2D/CMakeLists.txt

```diff
@@ -65,9 +64,10 @@ add_definitions(${PMacc_DEFINITIONS})
 
 
 ################################################################################
-# Find CUDA
+# Find cupla
 ################################################################################
-find_package(CUDA 7.5 REQUIRED)
+
+#FIND_PACKAGE("cupla" REQUIRED)
```

Comment:
This is removed because find cupla can only called once and is already called in pmaccConfig.cmake